### ISSUE TITLE
Stolen from and Banned.

### ISSUE THREAD

**Issue by Leonidas9798** [ObliqueNET/Stratos#325]
I was banned, for stealing from a previous town mate, which i did not do, i even offered to give their stuff back and they did not reply. until today, i had to deal with it, so we went into the house, which they were not a part of anymore, and gave them their stuff and more!, so now, lolskiz came to me with clarrissa, we found the house of megagamerZ, who kept flaunting that hes staff on terra along with his beanz friend. And accused me, of stealing so i said. " If i stole something i was unaware, and i will give it back." and i asked for him to show me what it was that i "stole" i gave the benefit of the doubt, theres a lot of people in the town, mabye i took down a house that i thought was vacant but happend not to be. This was not the case, i was messaged saying we need to talk by lolskins, i said alright, whats up mate?, he said apprently youve stolen something from a shulker box. I said if i have i did not intent to, and if i have i will return it. he told me that i stole something from someone that was a part of my town, someone who i had messaged telling them that if they wanted their stuff back to just tell me and ill give it to them, they messaged me a day later saying, "a simple no were also ok", that was not a i want my stuff, and it seemed like the opposite, just a no i was given, but i delayed taking down the house and clearing it, until about the fifth person came by and said they wanted the land, by that time i was already losing out on potential helpers money all these things, so i decided to clear some of the house with his seeming okay. its been 2 days, i cleared the stuff out the house that i seemingly gave to him, by this time it has been far past the time he had left the plot. I gave him resources for joining the town for as long as he was apart of it, and to build his house from a collected bunch of resources i keep for improving the town. i collected those and put them back into the town stock. now this morning 2 days after. he came to me saying"i want my stuff" so we went into his house and he said, i want that that that that that, i was like okay, you can have this stuff, (even though it was not his) i said if i break the healer it will be destroyed and he replied with "yes and which is why you will make me a new one" he has not been nice his whole time staying and now hes suddenly coming on with demands, but i leveled with him, i said what else do you want, he says the shulker box, i said okay, pretty sure i got it for him for the house but thats fine i have more then enough. Later i get the message from lolskins being accused of stealing. after i said i would return the stuff i asked for some information, he would not give it, i wanted to know what stuff, he would not give it, wanted to know who it was taken from and where he would not give it, for all i know it was just some kind of bullying or scam or being stolen from. then he said " cooperate with me your on thin ice as it is." thats a pretty aggressive statement for someone who just said " i did not know i was stealing i will give back everything i have taken if i did steal" for someone who is already cooperating to insinuate that they arent, and then saying they are on thin ice for, pretty much existing at this point i dont know what that loaded bias is supposed to be, im on thin ice, so there was already some kind of bias behind all of this happening. Against my standpoint. so i ask for some info agian, and i tell him "hey i am complying fully already theirs no need for that" but by this point its hard to take this situation seriously, so after some information exchanges, finding out that megagamez, who already is biased agianst me with Beanz whos last words were "your town is shit btw, did not join to be part of some random town im gonna join the staff for power you fools keep the shit" to which i could only respond with, "ok." so now, at this point after exchanging and telling lolskinz all the information i have about giving him the items and then telling them what was said about supposedly not wanting the stuff, i was told that even after giving more things to this person, they would have to take items from the whole city. That i spent the morning with other players terraforming and improving who are completely happy in the city and nice and not rude at all. Suddently we were in the storage room where i had to run around creating 3 sticks, give them not only the wood i already gave them but another stack of birch logs and all kids of wood, i had to scrable around take from the storage all these things, put them in my shulker box and give them to lolskinz so that he could steal them, and give them to someone who was not even online. And when i gave everything to him and more to give back like i said i would i was banned right after, when i was about to get back to work on the city.
![Screenshot (119)](https://user-images.githubusercontent.com/57264944/68043769-6e807e80-fcac-11e9-9b37-6cfb2ed282d5.png)
![Screenshot (120)](https://user-images.githubusercontent.com/57264944/68043770-6f191500-fcac-11e9-879a-f24ab00ce642.png)
![Screenshot (117)](https://user-images.githubusercontent.com/57264944/68043771-6f191500-fcac-11e9-805f-6700f5a41713.png)
![Screenshot (118)](https://user-images.githubusercontent.com/57264944/68043772-6f191500-fcac-11e9-8c55-e2e622159f13.png)
![Screenshot (186)](https://user-images.githubusercontent.com/57264944/68044042-11d19380-fcad-11e9-853e-b32d9adf332a.png)
![Screenshot (187)](https://user-images.githubusercontent.com/57264944/68044043-11d19380-fcad-11e9-9f9b-a7a6843cc476.png)
![Screenshot (188)](https://user-images.githubusercontent.com/57264944/68044044-11d19380-fcad-11e9-9530-1ee5fb91918d.png)
![Screenshot (189)](https://user-images.githubusercontent.com/57264944/68044046-11d19380-fcad-11e9-9c79-4f06a05e9048.png)
![Screenshot (190)](https://user-images.githubusercontent.com/57264944/68044047-126a2a00-fcad-11e9-9c3e-f553f76ab30c.png)
![Screenshot (191)](https://user-images.githubusercontent.com/57264944/68044048-126a2a00-fcad-11e9-82aa-86c21d5d94d7.png)
![Screenshot (192)](https://user-images.githubusercontent.com/57264944/68044049-126a2a00-fcad-11e9-8cf8-0b9f9306714f.png)
![Screenshot (121)](https://user-images.githubusercontent.com/57264944/68044050-126a2a00-fcad-11e9-8ea5-8145dfe9b440.png)
![Screenshot (122)](https://user-images.githubusercontent.com/57264944/68044051-126a2a00-fcad-11e9-944e-c42d5e5bea10.png)
![Screenshot (123)](https://user-images.githubusercontent.com/57264944/68044053-126a2a00-fcad-11e9-8ff4-8f43b60a5f47.png)
![Screenshot (124)](https://user-images.githubusercontent.com/57264944/68044054-1302c080-fcad-11e9-9a30-5164eb1eee11.png)
![Screenshot (125)](https://user-images.githubusercontent.com/57264944/68044055-1302c080-fcad-11e9-995a-306db70d8c7c.png)
![Screenshot (126)](https://user-images.githubusercontent.com/57264944/68044056-1302c080-fcad-11e9-991c-2f4617f4e391.png)
![Screenshot (127)](https://user-images.githubusercontent.com/57264944/68044057-1302c080-fcad-11e9-8277-bd4e69f66fc5.png)
![Screenshot (128)](https://user-images.githubusercontent.com/57264944/68044058-1302c080-fcad-11e9-804e-f3848cbe16a2.png)
![Screenshot (129)](https://user-images.githubusercontent.com/57264944/68044059-1302c080-fcad-11e9-829c-89eb52f2bad1.png)
![Screenshot (130)](https://user-images.githubusercontent.com/57264944/68044061-1302c080-fcad-11e9-8ab1-405853c83f82.png)
![Screenshot (131)](https://user-images.githubusercontent.com/57264944/68044062-1302c080-fcad-11e9-87d1-3c7a799ccf96.png)
![Screenshot (132)](https://user-images.githubusercontent.com/57264944/68044063-1302c080-fcad-11e9-92b9-3deec751f964.png)
![Screenshot (133)](https://user-images.githubusercontent.com/57264944/68044064-139b5700-fcad-11e9-96e1-aabd7139fea2.png)
![Screenshot (134)](https://user-images.githubusercontent.com/57264944/68044065-139b5700-fcad-11e9-84fd-42aac0ba9984.png)
![Screenshot (135)](https://user-images.githubusercontent.com/57264944/68044066-139b5700-fcad-11e9-8a0a-34bbb59ccf8c.png)
![Screenshot (136)](https://user-images.githubusercontent.com/57264944/68044068-139b5700-fcad-11e9-9ceb-a25cdcca1511.png)
![Screenshot (137)](https://user-images.githubusercontent.com/57264944/68044070-139b5700-fcad-11e9-803f-a74affa6c533.png)
![Screenshot (138)](https://user-images.githubusercontent.com/57264944/68044071-139b5700-fcad-11e9-8147-1e4ee0c3c915.png)
![Screenshot (139)](https://user-images.githubusercontent.com/57264944/68044072-139b5700-fcad-11e9-83aa-5d8352a5ecc1.png)
![Screenshot (140)](https://user-images.githubusercontent.com/57264944/68044074-1433ed80-fcad-11e9-90e6-d2faa6f69229.png)
![Screenshot (141)](https://user-images.githubusercontent.com/57264944/68044075-1433ed80-fcad-11e9-8aac-68cf9723f967.png)
![Screenshot (142)](https://user-images.githubusercontent.com/57264944/68044076-1433ed80-fcad-11e9-8165-c2255c019848.png)
![Screenshot (143)](https://user-images.githubusercontent.com/57264944/68044077-1433ed80-fcad-11e9-953a-70f77d194cc8.png)
![Screenshot (144)](https://user-images.githubusercontent.com/57264944/68044078-1433ed80-fcad-11e9-9f82-8059a145ceed.png)
![Screenshot (145)](https://user-images.githubusercontent.com/57264944/68044079-1433ed80-fcad-11e9-9481-a6d8bcaf8134.png)
![Screenshot (146)](https://user-images.githubusercontent.com/57264944/68044080-1433ed80-fcad-11e9-9fe5-605d359c31d9.png)
![Screenshot (147)](https://user-images.githubusercontent.com/57264944/68044082-1433ed80-fcad-11e9-9af4-4308ee915d6b.png)
![Screenshot (148)](https://user-images.githubusercontent.com/57264944/68044083-14cc8400-fcad-11e9-8e22-a4dad6f8a913.png)
![Screenshot (149)](https://user-images.githubusercontent.com/57264944/68044084-14cc8400-fcad-11e9-8279-724cb89c9e5b.png)
![Screenshot (150)](https://user-images.githubusercontent.com/57264944/68044085-14cc8400-fcad-11e9-8292-8efa364e675b.png)
![Screenshot (151)](https://user-images.githubusercontent.com/57264944/68044086-14cc8400-fcad-11e9-9743-094d033f20ae.png)
![Screenshot (152)](https://user-images.githubusercontent.com/57264944/68044087-14cc8400-fcad-11e9-881a-3915d167cbda.png)
![Screenshot (153)](https://user-images.githubusercontent.com/57264944/68044088-14cc8400-fcad-11e9-904f-b1e6fc7190b0.png)
![Screenshot (154)](https://user-images.githubusercontent.com/57264944/68044090-14cc8400-fcad-11e9-8504-316707f6c1ca.png)
![Screenshot (155)](https://user-images.githubusercontent.com/57264944/68044091-15651a80-fcad-11e9-86d1-07ca989476c8.png)
![Screenshot (156)](https://user-images.githubusercontent.com/57264944/68044092-15651a80-fcad-11e9-919f-b7cb0562e0c7.png)
![Screenshot (157)](https://user-images.githubusercontent.com/57264944/68044093-15651a80-fcad-11e9-9eac-183268d10b1b.png)
![Screenshot (158)](https://user-images.githubusercontent.com/57264944/68044094-15651a80-fcad-11e9-8f17-2e775c878b0b.png)
![Screenshot (159)](https://user-images.githubusercontent.com/57264944/68044095-15651a80-fcad-11e9-8492-2631ff801b44.png)
![Screenshot (160)](https://user-images.githubusercontent.com/57264944/68044096-15651a80-fcad-11e9-9137-ff138a687cd9.png)
![Screenshot (161)](https://user-images.githubusercontent.com/57264944/68044097-15651a80-fcad-11e9-81b9-fa86c3b41af1.png)
![Screenshot (162)](https://user-images.githubusercontent.com/57264944/68044098-15fdb100-fcad-11e9-9b21-64cdd0e803fa.png)
![Screenshot (163)](https://user-images.githubusercontent.com/57264944/68044100-15fdb100-fcad-11e9-85e8-365d2613d4f7.png)
![Screenshot (164)](https://user-images.githubusercontent.com/57264944/68044101-15fdb100-fcad-11e9-83da-15c25f36c4e5.png)
![Screenshot (165)](https://user-images.githubusercontent.com/57264944/68044102-15fdb100-fcad-11e9-9d7a-67e06ececc64.png)
![Screenshot (166)](https://user-images.githubusercontent.com/57264944/68044103-15fdb100-fcad-11e9-9c47-e948f6e74285.png)
![Screenshot (167)](https://user-images.githubusercontent.com/57264944/68044104-16964780-fcad-11e9-915b-20230b7a2cd6.png)
![Screenshot (168)](https://user-images.githubusercontent.com/57264944/68044105-16964780-fcad-11e9-874b-a66c895b60dc.png)
![Screenshot (169)](https://user-images.githubusercontent.com/57264944/68044106-16964780-fcad-11e9-82fb-54a0897c7afb.png)
![Screenshot (170)](https://user-images.githubusercontent.com/57264944/68044107-16964780-fcad-11e9-93ab-b9035d98d0ec.png)
![Screenshot (171)](https://user-images.githubusercontent.com/57264944/68044108-16964780-fcad-11e9-8a04-be2e577d84a0.png)
![Screenshot (172)](https://user-images.githubusercontent.com/57264944/68044109-16964780-fcad-11e9-9bb9-ee7d62a7d0c8.png)
![Screenshot (173)](https://user-images.githubusercontent.com/57264944/68044110-16964780-fcad-11e9-812a-e19ffe7a099e.png)
![Screenshot (174)](https://user-images.githubusercontent.com/57264944/68044112-172ede00-fcad-11e9-935c-09d5340e134a.png)
![Screenshot (175)](https://user-images.githubusercontent.com/57264944/68044113-172ede00-fcad-11e9-855c-68aafab5308d.png)
![Screenshot (176)](https://user-images.githubusercontent.com/57264944/68044115-172ede00-fcad-11e9-826b-c4792ae64774.png)
![Screenshot (177)](https://user-images.githubusercontent.com/57264944/68044116-172ede00-fcad-11e9-81e3-8b60884e4b21.png)
![Screenshot (178)](https://user-images.githubusercontent.com/57264944/68044117-172ede00-fcad-11e9-8020-7445d4284f52.png)
![Screenshot (179)](https://user-images.githubusercontent.com/57264944/68044118-172ede00-fcad-11e9-8771-6841b7dc8209.png)
![Screenshot (180)](https://user-images.githubusercontent.com/57264944/68044119-17c77480-fcad-11e9-8340-d51c4767354b.png)
![Screenshot (181)](https://user-images.githubusercontent.com/57264944/68044120-17c77480-fcad-11e9-89da-55cecd866cc8.png)
![Screenshot (182)](https://user-images.githubusercontent.com/57264944/68044121-17c77480-fcad-11e9-88fc-78af7be64723.png)
![Screenshot (183)](https://user-images.githubusercontent.com/57264944/68044123-17c77480-fcad-11e9-9867-163ea7d378ae.png)
![Screenshot (184)](https://user-images.githubusercontent.com/57264944/68044124-17c77480-fcad-11e9-95cc-964730cb74cc.png)
![Screenshot (185)](https://user-images.githubusercontent.com/57264944/68044125-17c77480-fcad-11e9-92d9-b4e75de3622c.png)
![Screenshot (197)](https://user-images.githubusercontent.com/57264944/68044278-64ab4b00-fcad-11e9-8226-48935ac0a656.png)
![Screenshot (198)](https://user-images.githubusercontent.com/57264944/68044279-6543e180-fcad-11e9-8409-9707a7521bff.png)
![Screenshot (199)](https://user-images.githubusercontent.com/57264944/68044280-6543e180-fcad-11e9-903d-d21b4ab8123f.png)
![Screenshot (200)](https://user-images.githubusercontent.com/57264944/68044281-6543e180-fcad-11e9-9c8d-3d71dde25135.png)
![Screenshot (201)](https://user-images.githubusercontent.com/57264944/68044282-6543e180-fcad-11e9-9835-8cbdf5122ece.png)
![Screenshot (202)](https://user-images.githubusercontent.com/57264944/68044283-6543e180-fcad-11e9-9784-221e084e745a.png)
![Screenshot (203)](https://user-images.githubusercontent.com/57264944/68044284-6543e180-fcad-11e9-9210-f5b3b043e526.png)
![Screenshot (204)](https://user-images.githubusercontent.com/57264944/68044285-6543e180-fcad-11e9-82ad-e7268bc17e95.png)
![Screenshot (193)](https://user-images.githubusercontent.com/57264944/68044287-65dc7800-fcad-11e9-83d1-66a21f853253.png)
![Screenshot (194)](https://user-images.githubusercontent.com/57264944/68044288-65dc7800-fcad-11e9-8c8a-20d745d4584b.png)
![Screenshot (195)](https://user-images.githubusercontent.com/57264944/68044289-65dc7800-fcad-11e9-92ee-d9c2542d5e56.png)
![Screenshot (196)](https://user-images.githubusercontent.com/57264944/68044290-65dc7800-fcad-11e9-8205-2a6253ab7daa.png)
![Screenshot (208)](https://user-images.githubusercontent.com/57264944/68044307-6ffe7680-fcad-11e9-8116-1c87f1a58690.png)
![Screenshot (205)](https://user-images.githubusercontent.com/57264944/68044308-6ffe7680-fcad-11e9-8131-3d1d05db8403.png)
![Screenshot (206)](https://user-images.githubusercontent.com/57264944/68044309-6ffe7680-fcad-11e9-9a7e-6ce07327c173.png)
![Screenshot (207)](https://user-images.githubusercontent.com/57264944/68044310-6ffe7680-fcad-11e9-8d22-d5fffa25b7f2.png)

**Comment by Grassip** [comment]
Hi there,

In this ban, our staff chose to give you a short tempban due to your cooperation when accused of stealing items. However, in this ban appeal, you lied about not actually stealing the items, when in fact you very obviously did. there is proof that you stole the items:
![image](https://user-images.githubusercontent.com/47408346/68046040-4b0c0280-fcb1-11e9-8a14-9ed21447a38b.png)
also, you wrongfully accused one of my staff members of stealing from _you_, which is outrageous, considering all he asked for was the items that you stole from your town member. 
![image](https://user-images.githubusercontent.com/47408346/68047177-177ea780-fcb4-11e9-8765-a9522052b999.png)
You also accused Bean of stating "your town is shit btw, did not join to be part of some random town im gonna join the staff for power you fools keep the shit" which is absolutely not true. Bean stated that she did say the first part of that statement, but then said she was going to join a more accepting town with staff in it. You then proceeded to kick her from the town before she could retrieve the rest of her stuff, to which she replied that you could just keep the rest of her things. You also provided no proof of her stating that despite having numerous screenshots of things that were going on before your ban.

You have spent this whole appeal trying to blame other people for something that you did. You have lied numerous times, wrongfully accused my staff members of doing things that they never did, and outright screwed yourself over by not just simply apologizing for stealing and being rude to our staff members. For that, instead of removing your tempban, I am going to permanently ban you for lying about our staff members, lying about your own actions, lying about other peoples statements, and for refusing to accept that you did something wrong.

Grassi

**Comment by Leonidas9798** [comment]
![Screenshot (247)](https://user-images.githubusercontent.com/57264944/68055780-d6908e00-fcc7-11e9-8c6c-3531c61c1128.png)
![Screenshot (248)](https://user-images.githubusercontent.com/57264944/68055938-43a42380-fcc8-11e9-99cc-92d2d23feafd.png)
![Screenshot (249)](https://user-images.githubusercontent.com/57264944/68055939-43a42380-fcc8-11e9-83ff-df23c838fd7b.png)
![Screenshot (253)](https://user-images.githubusercontent.com/57264944/68063936-aa870400-fceb-11e9-93ca-cedb03194eb6.png)
![Screenshot (254)](https://user-images.githubusercontent.com/57264944/68063971-fe91e880-fceb-11e9-80d3-85973d37e077.png)

![](https://cdn.discordapp.com/attachments/610476711393361946/639990556218490891/unknown.png)
![](https://cdn.discordapp.com/attachments/610476711393361946/639990424236195893/unknown.png)

![](https://cdn.discordapp.com/attachments/610476711393361946/639996969732931604/unknown.png)


"You also accused Bean of stating "your town is shit btw, did not join to be part of some random town im gonna join the staff for power you fools keep the shit" which is absolutely not true. Bean stated that she did say the first part of that statement, but then said she was going to join a more accepting town with staff in it." saying my statement is absolutely not true then stating that she admited to a part of it means that it held atleast some kind of merit, but heres a screenshot of them saying that since you thought i did not have it. And you saying im insinuating that a staff member is stealing from me rather then the person getting the items, very interesting. I never saw her say more accepting rather then 'they want me they need to calm down" so, it seems like your wrong, also check your facts i never kicked her from the town she left the discord and the town before i even got the chance to look up the command. No i am not trying to blame anyone i never said anyone was to fault i have a mere distaste for the lack of empathy and understanding for each other as fellow fans of the game. It was a simple trial where a lack of evidence was looked at because we could have came to the conclusion that there was no evidence because those coordinates to that biome where never in a plot a player owned, the spot those coordinates where in, was in the middle of a district that belonged to me that i did not finish building in the middle of the ocean where no houses where! and finnaly someone who would be rude to staff especially in this appeal, first asked you personally if it was okay to make the appeal and asked if you wanted to see it first, to make sure, who even asked if you were sure, a rude person generally does not do that especially when they feel wronged. 

    you have spent your whole appeal trying to put the blame on one person, when obviously a multitude of reasons and misunderstandings are at play, you have lied with beans about her statements, you have lied about accusations made from my side, you have accused me of being rude and accusing your staff, you have used un investigated evidence in your case, and you have pushed my appeal forward telling me it was okay to do so further cultivating the situation to escalate into something you can then put a final topping on yourself.  If anything is rude it is giving someone a ban reason of being a "fucking ignorant cunt." I have never lied, and you have lied. I have never lied about your staff memebers, you have lied about me with your staff members, you have lied about statements other people have made and my own. you have lied about my actions, and you refuse to accept that you have done something wrong and am putting the blame onto another being to hide the fact that you have not done your job properly. you will not get banned. you will not get temp banned, you will live out the rest of your life knowing that you can not work on your own to make proper decisions, you will live out knowing you can only make decisions based on the group you feel you fit into, and dont have the proper influence and morality, stability and leadership skills to be a lead. that is your punishment. I am not screwed over, ive gained much more then you, after this people came to me about the situation, feeling for me, telling me how good a person i was for even making a town to help new people, and that most people give up on noobs, i have gained much more then i have lost, and i have not lost a thing, I only dont have to be a part of something as corrupt as a server that does not allow for a sense of understanding, and cooperation. who thinks their staff is above the rules, and that staff is a title to be gained when you fill the requirement of either being a meme lord or abusing the system enough to actually get in, to fulfil your own gains. I have not screwed myself over, that would insinuate i have lost. i have not lost. But you have not gained, mabye you had nothing to lose already, but you defiantly have not gained anything useful that would make you better at your job, or as a person. Oblique is not a place for "- Friendliest Community/Staff" that people may have been looking for.

Our stuff:
- Survival/Adventure or Pure Survival
- No Pay-2-Win, so an actually balanced environment
- Friendliest Community/Staff<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
- Adventure map extending beyond into Survival with 10 custom-built towns!
- TMTrade - trade yo' TMs away for better ones
- Pixelmon world + Vanilla world (+ Nether and End)
- Quests for Legendaries and Town Lore
- Pokédex Ranks - get more claims and homes as you fill your PokéDex!
- RPG - McMMO oblique'd (catching, poketraining, archery, mining, +)
- 8 Player Gyms from level 25 to 100, on Normal and Advanced mode
- Sassy Discord bot that displays the server's chat
- InviteAFriend - do /invite [username] and get a Shiny, Masterball and Rare Candies!
- Trivia questions for cool rewards
- GriefPrevention (golden shovel claims) and Economy support (no thieves here, pls and ty)
- Basically months worth of custom code (My fingers still hurt)

- TWO servers for you to choose from: STRATOS (random legend spawning) and TERRA (quest legend spawning)!

Server IP: play.obliquemc.net
Website: http://obliquemc.com
Discord: Click me! Or don't; your call

**Comment by Leonidas9798** [comment]
Please make sure that you are aware of these rules. Joining Oblique will be taken as a sign of consent to these rules and guidelines.
 General Discord and Channels  
- Refrain from excessive usage of capital letters in chat. Do not caps lock.
- Curses like shit, fuck, damn, hell are allowed in general channels. Explicit, offensive or genuinely offensive ones are not. This is to Staff's discretion.
- Be mindful of the age of fellow players. Be respectful.
- Harassment, insulting, bullying and spamming of the chat and in private will be dealt with by a mute or a permanent ban.
- Advertisement must be pre-approved by Senior Staff. Please contact them about posting discord, youtube links or other forms of ads. Self-advertisement is included here. You may post your submissions for chatter in #streaming-lounge .
- Respect towards everyone, regardless of rank, is of utmost importance, always.
- Do not pursue Senior Staff regarding a pending application. It will be declined if you do.
- Pursuing or attempting to pursue any illegal activities as per Canadian legislation will result in a ban or tempban. Such activities include, without being limited to; hacking, identity theft, scamming, breaching of privacy and impersonation.

**Comment by fatyg5** [comment]
Thanks for taking the time to read our rules, many do not! Unfortunately, due to your breaking of our rules and the part about messaging rude things to other players regarding their towns, you will not be unbanned. Have a good day!
